<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim" respawn="true"/>
  <remap from="turtle1/cmd_vel" to="turtle1/cmd_vel_reversed"/>
  <remap from="turtle1/cmd_vel_reversed" to="turtle1/cmd_vel"/>
  <node pkg="turtlesim" type="turtle_teleop_key" name="teleop_key" required="true" launch-prefix="xterm -e"/>
  <node pkg="agitr" type="reverse_cmd_vel" name="reverse_velocity"/>
</launch>
